cytotable/convert.py

```diff
@@ -994,27 +994,17 @@ def convert(  # pylint: disable=too-many-arguments,too-many-locals
     dest_path: str,
     dest_datatype: Literal["parquet"],
     source_datatype: Optional[str] = None,
-    metadata: Union[List[str], Tuple[str, ...]] = cast(
-        list, config["cellprofiler_csv"]["CONFIG_NAMES_METADATA"]
-    ),
-    compartments: Union[List[str], Tuple[str, ...]] = cast(
-        list, config["cellprofiler_csv"]["CONFIG_NAMES_COMPARTMENTS"]
-    ),
-    identifying_columns: Union[List[str], Tuple[str, ...]] = cast(
-        list, config["cellprofiler_csv"]["CONFIG_IDENTIFYING_COLUMNS"]
-    ),
+    metadata: Optional[Union[List[str], Tuple[str, ...]]] = None,
+    compartments: Optional[Union[List[str], Tuple[str, ...]]] = None,
+    identifying_columns: Optional[Union[List[str], Tuple[str, ...]]] = None,
     concat: bool = True,
     join: bool = True,
-    joins: Optional[str] = cast(str, config["cellprofiler_csv"]["CONFIG_JOINS"]),
-    chunk_columns: Optional[Union[List[str], Tuple[str, ...]]] = cast(
-        list, config["cellprofiler_csv"]["CONFIG_CHUNK_COLUMNS"]
-    ),
-    chunk_size: Optional[int] = cast(
-        int, config["cellprofiler_csv"]["CONFIG_CHUNK_SIZE"]
-    ),
+    joins: Optional[str] = None,
+    chunk_columns: Optional[Union[List[str], Tuple[str, ...]]] = None,
+    chunk_size: Optional[int] = None,
     infer_common_schema: bool = True,
     drop_null: bool = True,
-    preset: Optional[str] = None,
+    preset: Optional[str] = "cellprofiler_csv",
```

Comment:
I'd definitely call out clearly in the docs that this has been changed from `None` to a default preset. At least be sure to update the docstring to indicate that this is the default value for `preset`; as of now it still says `None` is the default.

Reply:
Thank you! I've updated the documentation to reflect this. While this does change the preset default, the individual preset parameters were already coming from this preset (set from the parameter defaults as opposed to the preset logic within). My hope is that we can reach #58 at some point to avoid the use of preset options when one is not provided and instead dynamically choose options which make sense based on the data source(s).